src/hipscat/pixel_math/hipscat_id.py

```diff
@@ -0,0 +1,57 @@
+"""Compute the hipscat ID field"""
+
+import healpy as hp
+import numpy as np
+
+HIPSCAT_ID_HEALPIX_ORDER = 19
+
+
+def compute_hipscat_id(ra_values, dec_values):
+    """Compute the hipscat ID field.
```

Comment:
this should probably be in the google style of docstring like the rest of `pixel_math`, I think all you need to add as the `Args` and `Returns` sections